client/my-sites/themes/theme-showcase.jsx

```diff
@@ -506,50 +506,45 @@ class ThemeShowcase extends Component {
 							select={ this.onTierSelect }
 						/>
 					) }
-					{ isLoggedIn &&
-						( isNewSearchAndFilter ? (
-							<div className="theme__filters">
-								<ThemesToolbarGroup
-									items={ Object.values( this.tabFilters ) }
-									selectedKey={ this.state.tabFilter.key }
-									onSelect={ ( key ) =>
-										this.onFilterClick(
-											Object.values( this.tabFilters ).find(
-												( tabFilter ) => tabFilter.key === key
-											)
-										)
-									}
+					{ isNewSearchAndFilter && (
```

Comment:
The changes in this file are unrelated to the main purpose of this PR, but fixes an issue where logged-out showcase wouldn't display category filters in the new design.